assets/css/styles.css

```diff
@@ -0,0 +1,165 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  background-color: #fff4e3;
+  color: #816068;
+  display: flex;
+  flex-direction: column;
+  font-family: omnes-pro, Arial, Helvetica, sans-serif;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.no-shrink {
+  flex-shrink: 0;
+}
+
+.background {
+  z-index: -1;
+  pointer-events: none;
+}
+
+.background-top {
+  height: 136px;
+  overflow: hidden;
+  position: relative;
+}
+
+.background-bottom {
```

Comment:
Consider merging with the .background-top rule, and maybe rename more semantically (`.wave-container`?)

Reply:
I need 2 classes since I tweak them slightly differently below at different media points, but I renamed them to top/botttom-wave-container and combined these styles